From bb212194abf144dfe088e106934a885d6bab2649 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Fri, 25 Jan 2013 00:17:21 +0200
Subject: [PATCH] Remove enabling vertical interpolation (smoothing) in
 playback

When this feature is enabled, the graphics plane might get 45 degrees tilted.
For now disable this feature until complete investigation is done

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 xbmc/cores/VideoRenderers/DoveOverlayRenderer.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xbmc/cores/VideoRenderers/DoveOverlayRenderer.cpp b/xbmc/cores/VideoRenderers/DoveOverlayRenderer.cpp
index 72343ca..31278c6 100644
--- a/xbmc/cores/VideoRenderers/DoveOverlayRenderer.cpp
+++ b/xbmc/cores/VideoRenderers/DoveOverlayRenderer.cpp
@@ -204,7 +204,7 @@ bool CDoveOverlayRenderer::Configure(
     CLog::Log(LOGERROR, "%s::%s - Failed set source mode", CLASSNAME, __func__);
     return false;
   }
-  
+#if 0  
   GRAPHICS_SCALING scale = (GRAPHICS_SCALING) g_guiSettings.GetInt("videoscreen.graphics_scaling");
   if (scale == -1) scale=GR_SCALE_100;
   if (scale == GR_SCALE_100) /* Scaler is set differently when using graphics scaler */
@@ -216,7 +216,7 @@ bool CDoveOverlayRenderer::Configure(
       return false;
     }
   }
-
+#endif
   struct _sColorKeyNAlpha alpha;
 
   memset (&alpha, 0, sizeof(alpha));
-- 
1.7.9.5

